cli/main.c

```diff
@@ -240,14 +241,14 @@ static int gui(int argc, char **argv)
 		{"reset", 'r', "", CFG_NONE, &cfg.reset_bytes, no_argument,
 		 "reset byte counters"},
 		{"refresh", 'f', "", CFG_POSITIVE, &cfg.refresh, required_argument,
-		 "gui refresh period in seconds (default: 1 second)"},
+		 "GUI refresh period in seconds (default: 1 second)"},
```

Comment:
One of things I've always disliked was the name gui. 'monitor' or 'watch' would be better.... maybe a change for later though...

Reply:
I agree it could be named better, though I think this should be done later as "GUI" shows up in several places.